### PR TITLE
feat: how to configure substra clients

### PR DESCRIPTION
**Pull request by SdgJlbl**
How-to guide following https://github.com/Substra/substra/pull/353

### CODE REVIEW

**Review by RomainGoussault — COMMENTED** [review]
Thanks for the PR, it's very clear!

File: docs/source/documentation/client_configuration.rst [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+    client_2 = substra.Client(
+        client_name="org-2",
+        backend_type="remote",
+        url="https://org-2.com",
+    )
+
+
+Configuration using a configuration file
+----------------------------------------
+
```
Comment (ThibaultFy) [inline docs/source/documentation/client_configuration.rst:79]:
```suggestion
```

Comment (ThibaultFy) [reply docs/source/documentation/client_configuration.rst:79]: Don't want to jump line here

File: docs/source/documentation/client_configuration.rst [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+The last possibility for configuring a Substra client is to use a configuration YAML file.
+
+The configuration file contains a dictionary for each client you want to configure.
```
Comment (ThibaultFy) [inline docs/source/documentation/client_configuration.rst:82]:
```suggestion
The configuration file contains informations for each client you want to configure.
```

File: docs/source/documentation/client_configuration.rst [hunk 3]
```diff
@@ -0,0 +1,115 @@
+How-to configure Substra clients
+================================
+
+Whether you are using SubstraFL or directly the Substra SDK, you need to configure one ``Client`` by organisation,
+in order to register the datasets and the functions you want to use.
+
+This how-to guide exposes the different options you have to configure your clients. It targets both first-time and
+advanced Substra users.
```
Comment (ThibaultFy) [inline docs/source/documentation/client_configuration.rst:8]: What do you think to warn here on the order of priority ? IMO, this is quite error prone so we prefer to say it too many times than the opposite :)

**Review by ThibaultFy — COMMENTED** [review]
Thx its very clear !

**Review by ThibaultFy — APPROVED** [review]
Thx a lot !!

**Pull request merged**